pallets/task/src/lib.rs

```diff
@@ -292,11 +292,10 @@ pub mod pallet {
 			// Check that the extrinsic was signed and get the signer.
 			let signer = ensure_signed(origin)?;
 
```

Comment:
Could you maybe add a comment here to describe what is being checked in the line below? Seems to be the style for all other extrinsics.